tutorial/practicals/LFRic/building_code/1_simple_kernels/part1/README.md

```diff
@@ -168,8 +168,8 @@ to a single subroutine with all field arguments passed to the kernel calls:
 The name of the generated `invoke` subroutine in this case is just the
 `invoke` keyword with the appended numerical index. If there was a separate
 `invoke` call for each kernel, the subroutine name would also contain the
-name of the called kernel (see [PSyclone API](
-https://psyclone.readthedocs.io/en/stable/algorithm_layer.html#api)
+name of the called kernel (see [PSyclone Algorithm-layer](
+https://psyclone.readthedocs.io/en/stable/introduction_to_psykal.html#algorithm-layer)
```

Comment:
The new links actually give me a 404 if I try to visit them in a web browser. They should be `latest` rather than `stable`. This does then beg the question as to why the link checks are passing? Please could you investigate?

Reply:
The output from the linkchecker looks a bit suspect (sparse):
![image](https://github.com/stfc/PSyclone/assets/17409212/589cc0ad-f990-46c8-be30-30cd97dbd6b1)

Reply:
@arporter I first started with /latest/ but then realised that everything else was /stable/ in the tutorials and the github workflow replaces them to /latest/ with the `.github/workflows/mlc_config.json` for checking the links. So what is the policy here, is stable ok considering they will appear in the next release? Or should I still update them to latest and at some point in the future after the release we update them to stable?

Reply:
Aha! In that case I think you've done the right thing - that must be why we setup the mlc_config.json that way.